src/parser.cpp

```diff
@@ -337,6 +337,8 @@ void Parser::ParserImpl::loadModel(const ModelPtr &model, const std::string &inp
                 err->setRule(SpecificationRule::MODEL_CHILD);
                 mParser->addError(err);
             }
+        } else if (childNode->isType("comment")) {
```

Comment:
From this I take it we're happy to read XML into objects without retaining references to the original XML?

So for example, code that's been "through" libCellML will have lost all comments and might have elements in a different order then the input?

(This would be fine by me)

Reply:
Hmm, some model developers actually have quite a few comments in their CellML file. This is, in fact, the reason we added support for comments in OpenCOR's `CellML Text` view. However, I can see that being somewhat difficult using our current approach...